pages/common/sops.md

```diff
@@ -29,4 +29,4 @@
 
 - Show the difference between two `sops` files:
 
-`diff <(sops -d {{path/to/secret1.enc.yaml}}) <(sops -d {{path/to/secret2.enc.yaml}})`
+`diff {{path/to/sops_file1}} {{path/to/sops_file2}}`
```

Comment:
Diff doesn't work for encrypted files. The `sops -d` command must be called to decrypt the sops files.

Reply:
Would replacing the placeholder for `{{decrypted_sops_file}}` cut it?

Reply:
In my opinion not, because decrypted sops files are just plain json or yaml files if I understand the docs of sops correctly. 

Also, I do not think that we should eliminate process substitution from our examples. I highly doubt that any user uses a shell that doesn't support process substitution as their daily driver (no **beginner** user, anyway). I think that tldr pages remain more useful with the process substitution examples left in.

Reply:
> In my opinion not, because decrypted sops files are just plain json or yaml files if I understand the docs of sops correctly.
> 
> Also, I do not think that we should eliminate process substitution from our examples. I highly doubt that any user uses a shell that doesn't support process substitution as their daily driver (no **beginner** user, anyway). I think that tldr pages remain more useful with the process substitution examples left in.


*SOPS is an editor of encrypted files that supports YAML, JSON, ENV, INI and BINARY formats [...]*
from [here](https://github.com/getsops/sops#sops-secrets-operations)
so the decrypted `sops` file is in one of the formats above.